build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         maven { url "https://plugins.gradle.org/m2/" }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:8.6.0'
+        classpath 'com.android.tools.build:gradle:8.5.2'
```

Comment:
What's the purpose of this ?

Reply:
Without a strong reason, I request previous one

Reply:
Well the 8.6.0 was not working with me and it said to change to a stable version i.e. 8.5.2  

But now i have changed it back to original.